tools/rosetta/README.md

```diff
@@ -57,6 +57,14 @@ simd rosetta
      --addr "rosetta binding address (ex: :8080)"
 ```
 
+## Use Rosetta Standalone
+
+To use Rosetta standalone, without having to add it in your application, install it with the following command:
+
+```bash
+go install cosmossdk.io/tools/rosetta/cmd/rosetta
```

Comment:
Unfortunately, this will not work until https://github.com/coinbase/rosetta-sdk-go/issues/449 is resolved.

Reply:
Added!